inginious/frontend/task_dispensers/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 
 class TaskDispenser(metaclass=ABCMeta):
-    def __init__(self, task_list_func, dispenser_data):
+    def __init__(self, task_list_func, dispenser_data, database=None, course_id=None):
```

Comment:
Making those arguments optional is probably not needed here and has probably no much sense as they will always be passed by design (line 86 in ``inginious/frontend/courses.py``).

The new arguments are also missing from the docstring.